packages/v4/patternfly-docs/content/accessibility/breadcrumb/breadcrumb.md

```diff
@@ -3,13 +3,70 @@ id: Breadcrumb
-Most frequently, breadcrumb consists of many links which will use **Tab** to navigate through them, **Shift + Tab** to go backwards, and **Enter** to activate each link. 
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
+## Accessibility
 
-In general, the breadcrumb component already has accessibility built in. However, if you would like to customize it, edit the following:
+To implement an accessible PatternFly **breadcrumb** component:
+
+- Give each item in the breadcrumb concise, descriptive text
+- Ensure each breadcrumb item can be navigated to and interacted with via keyboard and other assistive technologies such as a screen reader
+- Provide context to notify users if a breadcrumb item will open in a new tab or window
+- Provide an `aria-label` to the breadcrumb if there are multiple `nav` elements or breadcrumb components on the page
+- Follow the accessibility documentation for any other components used within a breadcrumb, such as a [dropdown](/components/dropdown) used within a breadcrumb item
+
+For the PatternFly React library:
+
+- Pass in the `isActive` prop to the breadcrumb item whose page is currently active
+
+For the HTML/CSS library:
+
+- Pass in the `aria-current="page"` attribute to the breadcrumb item whose page is currently active
+
+## Testing
+
+At a minimum, a breadcrumb should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-1" label="Each breadcrumb item has descriptive text." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-2" label="Standard keyboard navigation can be used to navigate between each breadcrumb item or other focusable elements." description={<span><kbd>Tab</kbd> navigates to the next breadcrumb item or focusable element, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous breadcrumb item or focusable element.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-3" label="Users are able to tell whether a breadcrumb item will open in a new tab or window." description={<span>This can be best achieved by rendering an icon for visual context, and passing in visually-hidden text via the <code className="ws-code">pf-screen-reader</code> class.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-4" label="Users are able to tell whether a breadcrumb item will open in a new tab or window." description={<span>This can be best achieved by rendering an icon for visual context, and passing in visually-hidden text via the <code className="ws-code">pf-screen-reader</code> class.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-5" label={<span>The breacrumb has an <code className="ws-code">aria-label</code> if there are other <code className="ws-code">nav</code> elements or breacrumbs on the page.</span>} description="This helps differentiate the various navigations to users of assistive technologies." />
```

Comment:
Typo in "breadcrumb"
"Various navigations" sounds funny to me - maybe "This helps users of assistive technologies to differentiate between multiple navigational elements." ?